traefik/templates/_helpers.tpl

```diff
@@ -185,7 +185,9 @@ It requires a dict with "Version" and "Hub".
         {{- $hubProxyVersion = "v3.1" }}
      {{- else if semverCompare "<v3.10.2-0" $version -}}
         {{ $hubProxyVersion = "v3.2" }}
-     {{- else if semverCompare "<v3.17.0-0" $version -}}
+     {{- else if semverCompare "<v3.15.3-0" $version -}}
+        {{ $hubProxyVersion = "v3.3" }}
+     {{- else if or (semverCompare "=v3.16.0" $version) (semverCompare "=v3.16.1" $version) -}}
```

Comment:
```suggestion
     {{- else if semverCompare "<v3.16.2-0" -}}
```

Reply:
You're right :+1: . 
It seems it's possible to compare multiple version in the same expr, like ">= 1.2 < 3.0.0 || >= 4.2.3" ([source](https://helm.sh/docs/chart_template_guide/function_list/#basic-comparisons))
I'll close this suggestion and make a new one. 

Do you think you can add tests, too ? It seems required now that we are introducing this level of complexity.